nff/tests/dynamics_test.py

```diff
@@ -15,8 +15,8 @@
 from ase.io.trajectory import Trajectory
 from ase import Atoms
 
-from nff.md.utils import mol_dot, mol_norm, ZhuNakamuraLogger, atoms_to_nxyz
-from nff.md.nvt_test import NoseHoover, NoseHooverChain
+from nff.md.utils_ax import mol_dot, mol_norm, ZhuNakamuraLogger, atoms_to_nxyz
+from nff.md.nvt_ax import NoseHoover, NoseHooverChain
```

Comment:
I suspected something like we have already discussed concerning deprecated code might be going on with these `_ax` files. I anyway changed the import here, because
 1) some of the imported objects do not exist in the files, so the current import statement is broken
  2) it is just a test, so we do not change actual production runs.
  
If this test ever makes any issues, because the `_ax` files make some problems or we remove them, I would say we can look into removing this test or adapting it to the current code base?